orion/primitives/timeseries_postprocessing.py

```diff
@@ -0,0 +1,191 @@
+import operator
+import numpy as np
+import pandas as pd
+
+def extract_dimension(X, dim=None):
+    """
+    The function checks if the dataset being used is valid i.e has a length greater than 0 and contains the dimension required
+    Args:
```

Comment:
add space between the description, arguments and returns. Here is the general guildeline to follow:

```
"""Short description in a single line ending with a dot.

Longer description that can span across multiple lines. Longer
description that can span across multiple lines. Longer description
that can span across multiple lines.

Args:
    arg_name (arg_type):
        argument description.
    arg_name (arg_type):
        Argument description that spans across multiple lines. Argument
        description that spans across multiple lines. Argument description
        that spans across multiple lines.

Returns:
    return_type:
        description of the returned objects
```